migration-util/src/main/java/edu/unc/lib/dcr/migration/content/ContentObjectTransformer.java

```diff
@@ -459,6 +459,8 @@ private Resource getResourceType(Resource bxc3Resc) {
         if (contentModels.contains(COLLECTION.getResource())) {
             if (Cdr.AdminUnit.equals(parentType) || !options.isTopLevelAsUnit()) {
                 return Cdr.Collection;
+            } else if (Cdr.Collection.equals(parentType)) {
```

Comment:
I don't think this is quite going to work as expected. When we migrate the RLA the `isTopLevelAsUnit` flag is going to be `false` in practice, so I think the first `if` is always going to win for Collections at the moment. Swapping their order would probably work. 

Additionally, the migration util doesn't seed the parentType parameter with the type of the bxc5 destination. This might be a problem if we end up migrating the RLA or other collections like it in multiple deposits, since it won't be aware the subcollections are going into a collection. It would be prudent to start doing this now, i'll create another ticket for that.

Reply:
Also, could you add a warn level log statement with the pid of the object being changed from a collection to a folder?

Reply:
https://jira.lib.unc.edu/browse/BXC-2845